api/templates/model_enum.mustache

```diff
@@ -0,0 +1,75 @@
+// {{{classname}}} {{#description}}{{{.}}}{{/description}}{{^description}}the model '{{{classname}}}'{{/description}}
```

Comment:
Can you add a line to `api/templates/README.md` describing the change added to this template?

Reply:
Done